src/lib/y2storage/proposal/space_maker.rb

```diff
@@ -413,13 +470,14 @@ def delete_disk_content(planned_partitions, lvm_helper, disk)
         log.info "BEGIN delete_disk_content with disk #{disk}"
 
         disks_for(new_graph, disk).each do |dsk|
-          break if success?(planned_partitions)
           log.info "Checking if the disk #{dsk.name} has a partition table"
 
-          if dsk.has_children? && dsk.partition_table.nil?
-            log.info "Found something that is not a partition table"
-            remove_content(dsk, lvm_helper)
-          end
+          next unless dsk.has_children? && dsk.partition_table.nil?
+
+          log.info "Found something that is not a partition table"
+          remove_content(dsk, lvm_helper)
+
+          break if success?(planned_partitions)
```

Comment:
btw, now this success is called only when content is removed. Is it intention?

Reply:
yes, only when we changed something

Reply:
ok, I just want to be sure.